radio/src/targets/common/arm/stm32/usbd_cdc.cpp

```diff
@@ -230,7 +230,13 @@ static uint16_t VCP_DataRx (uint8_t* Buf, uint32_t Len)
     }
   }
 #endif
-
+//OW
+#if defined(TELEMETRY_MAVLINK) && defined(USB_SERIAL)
+  for (uint32_t i = 0; i < Len; i++) {
+    mavlinkTelemUsbRxFifo.push(Buf[i]);
```

Comment:
Please use the provided queue USB serial (see CLI) and make it bigger if required for MAVLink.